templates/community/index.hbs

```diff
@@ -82,6 +82,29 @@
         </ul>
       </div>
     </div>
+    <div class="dn db-l">
```

Comment:
The original code prior to 11 days ago had `.dn-m` on line 86 the problem is that that the flex class overrides that `.dn-m` for specificity reasons. So it needs to sit higher in its own container.

So the rule on line 85 now says `display: none` and `display:block on large screen`.